src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    public static void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> operationSums = new HashMap<>();
+
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
You should use try-with-resources for FileReader as well to ensure that all resources are closed properly.